### PR TITLE
feat:expenselistitems

### PR DESCRIPTION
**Pull request by yusufshaikhnag**

### CODE REVIEW

File: components/ExpensesOutput/ExpensesOutput.js [hunk 1]
```diff
@@ -33,6 +33,24 @@ const DUMMY_EXPENSES = [
         description: 'Another book',
         amount: 18.59,
         date: new Date('2022-02-18')
+      },
```
Comment (ifarhanpatel) [inline components/ExpensesOutput/ExpensesOutput.js]: Create a separate file for dummy data.
 eg. Expenses.mock.js and import.

File: components/ExpensesOutput/ExpenseItem.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import { Pressable, Text, View, StyleSheet } from "react-native";
+import { GlobalStyles } from "../../constants/styles";
+import { getFormattedDate } from "../../util/date";
+
+function ExpenseItem({ description, amount, date }) {
+    function expensePressHandler() {}
```
Comment (ifarhanpatel) [inline components/ExpensesOutput/ExpenseItem.js]: Needs to be wrapped with useCallback() hook. Also try using arrow functions for declaring functions.

File: components/ExpensesOutput/ExpensesList.js [hunk 3]
```diff
@@ -1,7 +1,8 @@
-import { FlatList, Text } from "react-native";
+import { FlatList } from "react-native";
+import ExpenseItem from "./ExpenseItem";
 
 function renderExpenseItem(itemData) {
```
Comment (ifarhanpatel) [inline components/ExpensesOutput/ExpensesList.js:4]: Needs to be wrapped with useCallback() hook.

**Review by ifarhanpatel — CHANGES_REQUESTED** [review]
Also add eslint to the project.

File: components/ExpensesOutput/ExpenseItem.js (hunk 2 above)
Comment (yusufshaikhnag) [reply components/ExpensesOutput/ExpenseItem.js]: Ok

**Pull request merged**